viz_v2/wiki_categorize/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can you add a README to this directory to explain what this npm project is about?